constants.py

```diff
@@ -3,8 +3,20 @@
 """
 
 import os
+import thousandaire.trading_cost
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
 DATA_DIR = os.path.join(ROOT_DIR, 'data')
 DATA_LIST_ALL = ['currency_price_tw', 'workdays']
+TRADING_INSTRUMENTS = {
+    ('currency', 'TW'):
+        ('USD', 'JPY', 'AUD', 'SEK', 'NZD', 'EUR', 'ZAR', 'GBP')
```

Comment:
Is it possible to cover all currencies avaiable from Yushan bank?
For (currency, TW), Yushan may be the most tradable broker.
https://www.esunbank.com.tw/bank/personal/deposit/rate/forex/foreign-exchange-rates

Besides, can we order the list by [their trading volumn](https://en.wikipedia.org/wiki/Currency_pair#Base_currency)?
Whatever the trading market is (stock, future, currency...), the most tradable instruments are always those with top volumns. Sorting the list by such order can guarantee that, if we want to remove some instrument one day, it is more possible to be at tail, instead of some middle position, which will affect our alphas less.